executor/api/rabbitmq/connection.go

```diff
@@ -76,7 +76,13 @@ func (c *connection) Close() error {
 }
 
 // implements retry logic with delays for establishing AMQP connections.
-func (c *connection) connect() error {
+// prefetchCnt is variadic since Go does not support function argument default value
+func (c *connection) connect(prefetchCnt ...int) error {
```

Comment:
can we just hardcode pfc to 1 if we aren't passing a different value anywhere?

Reply:
makes sense, I think we could just add it back when/if we need to specify a different value

Reply:
Okay, then I will revert to previous version